exercise11.rb

```diff
@@ -0,0 +1,33 @@
+require 'selenium-webdriver'
+
+class Exercise
+
+    def setup
+        Selenium::WebDriver::Firefox::Service.driver_path = "/home/vibhav/Desktop/INFUSE/driver/geckodriver"
+        @driver = Selenium::WebDriver.for :ff
+        @url = "https://the-internet.herokuapp.com/tables"
+    end
+
+    def action(val)
+        begin
+            @driver.get(@url)
+           
+            select = @driver.find_elements(:tag_name, "td")
```

Comment:
Better way would be getting row and column index

Reply:
Didn't understand this comment